modules/develop/pages/transactions.adoc

```diff
@@ -19,18 +19,19 @@ By default, the `enable_transactions` cluster configuration property is set to t
 
 The required `transactional.id` property acts as a producer identity. It enables reliability semantics that span multiple producer sessions by allowing the client to guarantee that all transactions issued by the client with the same ID have completed prior to starting any new transactions.
 
-[NOTE]
-====
-* A transaction can span partitions from different topics, and a topic can be deleted while there are active transactions on one or more of its partitions. In-flight transactions can detect deletion events, remove the deleted partitions (and related messages) from the transaction scope, and commit changes to the remaining partitions.
-* Ongoing transactions can prevent consumers from advancing. To avoid this, don't set transaction timeout (`transaction.timeout.ms` in Java client) to high values: the higher the timeout, the longer consumers may be blocked. By default, it's about a minute, but it's a client setting that depends on the client.
-* When upgrading, make sure to use maintenance mode with xref:manage:cluster-maintenance/rolling-upgrade.adoc[rolling upgrades].
-====
-
 The two primary use cases for transactions are:
 
 * xref:develop:transactions.adoc#atomic-publishing-of-multiple-messages[Atomic (all or nothing) publishing of multiple messages]
 * xref:develop:transactions.adoc#exactly-once-stream-processing[Exactly-once stream processing]
 
+=== Transaction usage tips
+
+* A transaction can span partitions from different topics, and a topic can be deleted while there are active transactions on one or more of its partitions. In-flight transactions can detect deletion events, remove the deleted partitions (and related messages) from the transaction scope, and commit changes to the remaining partitions.
+* Ongoing transactions can prevent consumers from advancing. To avoid this, don't set transaction timeout (`transaction.timeout.ms` in Java client) to high values: the higher the timeout, the longer consumers may be blocked. By default, it's about a minute, but it's a client setting that depends on the client.
+* When upgrading, make sure to use maintenance mode with xref:manage:cluster-maintenance/rolling-upgrade.adoc[rolling upgrades].
```

Comment:
You could instead make rolling upgrade a [glossterm link](https://deploy-preview-243--redpanda-docs-preview.netlify.app/current/reference/glossary/#rolling-upgrade) or have the link go direct to the [rolling upgrade section](https://deploy-preview-243--redpanda-docs-preview.netlify.app/current/upgrade/rolling-upgrade/#perform-a-rolling-upgrade)?